modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -313,6 +314,23 @@ public void testDeleteObjectRelationship() {
 						objectRelationship.getDeletionType(), true,
 						objectRelationship.getLabelMap()));
 			});
+
+		ObjectRelationship systemObjectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition2.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				StringUtil.randomId(), true,
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		AssertUtils.assertFailure(
+			ObjectRelationshipSystemException.class, false,
+			"Only allowed bundles can delete system object relationships",
+			() -> _objectRelationshipLocalService.deleteObjectRelationship(
+				systemObjectRelationship));
```

Comment:
AssertUtils.assertFailure() is ordered by the Exception class name.
Then `ObjectRelationshipReverseException.class` comes first than `ObjectRelationshipSystemException.class`. 
Could you reorder please?

Reply:
In order to do this, you will have to move the block of code (adding and deleting part) after the `AssertUtils.assertFailure(ObjectRelationshipReverseException.class,...)`.

Reply:
After the `AssertUtils.assertFailure(ObjectRelationshipSystemException.class,...)`, can you properly delete the `systemObjectRelationship` please?
It is a good practice to delete variable used in tests :+1: